web/content/docs/tools/workflows/create-a-simple-parallel-model/index.pandoc

```diff
@@ -41,12 +41,30 @@ where `a`, `b`, and `c` should be choosen according to the needs.
 |  422  |  422  |  422  | ~ 75.15 | 240 | yes |
 |  465  |  465  |  465  | ~ 100.54 | 320 | |
 
+
+For the boundary conditions, if they are simple (_i.e._ homogeneous), the
+simplest `.gml` file is sufficient for the serial case, but for heterogeneous
+boundary conditions and parallelization boundary mesh files are needed. There
+are two possibilities to create such files:
+
+ - If there is a `.gml` file, use [`constructMeshesFromGeometry`
+   tool](../../model-preparation/constructmeshesfromgeometry/), which takes the
```

Comment:
For links you may use Hugo's [ref](https://gohugo.io/content-management/shortcodes/#ref-and-relref)-shortcode, e.g.:

```
[`constructMeshesFromGeometry` tool]({{< ref "model-preparation/ConstructMeshesFromGeometry" >}})
```

Reply:
Updated.
I was told by Lars, that this type of links is checked for validity while the site is being build.